File: frontend/src/app/models/Session.ts
export class Session {
  id: number;
  mode: string;
  date: string;
  time: string;

  constructor(id: number, date: string, time: string, mode: string) {
    this.id = id;
    this.date = date;
    this.time = time;
    this.mode = mode;
  }
}

File: fapi/src/main/java/com/netcracker/cinema/fapi/model/Session.java
package com.netcracker.cinema.fapi.model;

import java.sql.Date;
import java.sql.Time;

public class Session {
    private Long id;
    private Date date;
    private Time time;
    private String mode;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public Time getTime() {
        return time;
    }

    public void setTime(Time time) {
        this.time = time;
    }

    public String getMode() {
        return mode;
    }

    public void setMode(String mode) {
        this.mode = mode;
    }
}
